content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,85 @@ further_reading:
 
-The .NET Tracer can instrument the following libraries automatically:
-
-| Framework or library            | NuGet package                                                           | Integration Name     |
-|---------------------------------|-------------------------------------------------------------------------|----------------------|
-| ASP.NET Core                    | `Microsoft.AspNetCore`</br>`Microsoft.AspNetCore.App`</br>2.0+ and 3.0+ | `AspNetCore`         |
-| ADO.NET                         | `System.Data.Common`</br>`System.Data.SqlClient` 4.0+                   | `AdoNet`             |
-| HttpClient / HttpMessageHandler | `System.Net.Http` 4.0+                                                  | `HttpMessageHandler` |
-| WebClient / WebRequest          | `System.Net.Requests` 4.0+                                              | `WebRequest`         |
-| Redis (StackExchange client)    | `StackExchange.Redis` 1.0.187+                                          | `StackExchangeRedis` |
-| Redis (ServiceStack client)     | `ServiceStack.Redis` 4.0.48+                                            | `ServiceStackRedis`  |
-| Elasticsearch                   | `Elasticsearch.Net` 5.3.0+                                              | `ElasticsearchNet`   |
-| MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
-| PostgreSQL                      | `Npgsql` 4.0+                                                           | `AdoNet`             |
-| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                | `RabbitMQ`           |
-| Service Fabric Remoting         | `Microsoft.ServiceFabric.Services.Remoting` 4.0.470+                    | `ServiceRemoting`    |
-
-<div class="alert alert-info">
-<strong>Note:</strong> The ADO.NET integration instruments calls made through the <code>DbCommand</code> abstract class or the <code>IDbCommand</code> interface, regardless of the underlying implementation. It also instruments direct calls to <code>SqlCommand</code>.
-</div>
+| Processor architectures                                                                                            |
+| ------------------------------------------------------------------------------------------------------------------ |
+| Windows x86 (`win-x86`)                                                                                            |
+| Windows x64 (`win-x64`)                                                                                            |
+| Linux x64 (`linux-x64`)                                                                                            |
+| Alpine Linux x64 (`linux-musl-x64`)                                                                                |
+| Linux ARM64 (`linux-arm64`)<br><br>Added in version 1.27.0, automatic instrumentation only supported on .NET 5.    |
```

Comment:
It wasn't clear before, but this entire list only applies to automatic instrumentation (manual instrumentation is processor-agnostic). Calling it out like this is misleading. I think we should add a caption to the table, and this note can be simplified to ".NET 5 only, added in version 1.27.0."

Reply:
Added a note and updated Linux Arm64 comment. Will be reflected in new PR.

Reply:
Updated and reflected in: https://github.com/DataDog/documentation/pull/11200/commits/77793e5a94b644a2679fb90ab651acd9a154876b